docs/source/api/user-guide/polars/expressions/manipulation.ipynb

```diff
@@ -514,6 +514,40 @@
     "OpenDP enforces that inputs to `.fill_nan` are row-by-row."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Replace\n",
+    "Useful for recoding. \n",
```

Comment:
This might be out of scope for what you had envisioned, but would it be worth while to include examples of all the forms that you include in the tests? I don't know that we can expect users to bounce over to the Polars docs for details.

Reply:
I have a dictionary example with `replace_strict` now, and some more explanatory text.